kickstarts/partials/post/vagrant.ks.erb

```diff
@@ -0,0 +1,23 @@
+# Add a user "vagrant" with password "vagrant" and a well known public key.
+# Note this is how Vagrant works. You are assumed not to use a Vagrant box on
+# a publicly acessible network.
```

Comment:
See: https://www.vagrantup.com/docs/boxes/base.html Vagrant needs to be able to log in to virtual machines, and for that it requires the insecure key pair, and default password. All vagrant boxes work like this.

Note that Vagrant *does* create a box behind a  NAT with no inbound access by default.  You have to do real work in order to expose it publicly.